ChatGPT/legos/chatgpt_create_chat/chatgpt_create_chat.py

```diff
@@ -0,0 +1,52 @@
+from __future__ import annotations
+
+##
+##  Copyright (c) 2021 unSkript, Inc
+##  All rights reserved.
+##
+from pydantic import BaseModel, Field
+from typing import Dict, List
+
+
+
+from typing import List
+
+
+
+
+class InputSchema(BaseModel):
+    messages: List = Field(
+        ..., description='The messages for Chat GPT', title='messages'
+    )
+    model: str = Field(
+        ..., description='The ChatGPT Model to use in your query.', title='model'
+    )
+    temperature: float = Field(
+        ...,
+        description='The Temperature of the response.  Valid values are 0 to 2. Higher values will give more random results.',
+        title='temperature',
+    )
+
+
+def chatgpt_create_chat(handle, model: str, messages:list, temperature:float)-> Dict:
+    response = handle.ChatCompletion.create(
```

Comment:
Shouldnt the messages be like
```
    messages = [{"role": "user", "content": prompt}]
```
where prompt is nothing but your messages input, as the user will not provide the prompt.